plaso/cli/helpers/event_filters.py

```diff
@@ -108,17 +109,22 @@ def ParseOptions(cls, options, configuration_object):
 
     time_slice_event_timestamp = None
     if time_slice_event_time_string:
-      # Note self._preferred_time_zone is None when not set but represents UTC.
-      preferred_time_zone = getattr(
-          configuration_object, '_preferred_time_zone', None) or 'UTC'
-      timezone = pytz.timezone(preferred_time_zone)
-      time_slice_event_timestamp = timelib.Timestamp.FromTimeString(
-          time_slice_event_time_string, timezone=timezone)
-      if time_slice_event_timestamp is None:
+      if ' ' in time_slice_event_time_string:
         raise errors.BadConfigOption(
-            'Unsupported time slice event date and time: {0:s}'.format(
+            'Time slice date and time must be defined in ISO 8601 format.')
```

Comment:
Please add an example of what an ISO 8601 formatted date and time looks like to this error message.